pyproject.toml

```diff
@@ -3,10 +3,10 @@
 [tool.codespell]
 # note: pre-commit passes explicit lists of files here, which this skip file list doesn't override -
 # this is only to allow you to run codespell interactively
-skip = "./pyproject.toml,./.git,./.github,./cpp/build,.*egg-info.*,./.mypy_cache,./benchmarks/utilities/cxxopts.hpp"
+skip = "./.git,./build,./python/_skbuild/,.*egg-info.*,./.mypy_cache,./pyproject.toml,./benchmarks/utilities/cxxopts.hpp"
 # ignore short words, and typename parameters like OffsetT
 ignore-regex = "\\b(.{1,4}|[A-Z]\\w*T)\\b"
-ignore-words-list = "inout,thirdparty"
+ignore-words-list = "thirdparty,couldn"
```

Comment:
I understand the other stuff, but what's going on here? I'm approving regardless.

Reply:
There's a bug in codespell that incorrectly marks `couldn't` as an invalid word. Several projects are working around this. I tried to report it here on an issue I thought was relevant, but it was marked off-topic by the maintainer. https://github.com/codespell-project/codespell/issues/2786#issuecomment-2543379509